lib/src/summarize.dart

```diff
@@ -4,23 +4,235 @@
 
 library dart_pad.summarize;
 
+import 'package:crypto/crypto.dart';
+
 import '../services/dartservices.dart';
 
 /// Instances of this class take string input of dart code as well as an analysis result,
 /// and output a text description ofthe code's size, packages, and other useful information.
 class Summarizer {
   _SummarizeToken storage;
-
+  String _code;
```

Comment:
You can make these three fields `final` and public (no underscore). Perhaps change `_code` to `dart`?